src/components/NavBar.vue

```diff
@@ -1,5 +1,5 @@
-<template>
-	<h1 color-white text-left>{{ msg }}</h1>
+<template>\
```

Comment:
Codacy has a fix for the issue: [Insert `⏎↹`](https://app.codacy.com/gh/SonodaKazuto/stelo/pullRequest?prid=9743489)
```suggestion
<template>
```